### ISSUE TITLE
Transform all bullets instead of just the last one for org-indent-mode?

### ISSUE THREAD

**Issue by dakrone** [sabof/org-bullets#15]
It would be nice if `org-bullets` transformed all of the bullets into UTF-8 instead of just the visible one, for instance:

![screenshot from 2015-12-24 09-37-01](https://cloud.githubusercontent.com/assets/19060/11996535/110be9be-aa22-11e5-96f6-a3ed817508c6.png)

And then, when I copy and paste the block to send to someone, I get a mixture of regular (`*`) stars and UTF-8 stars in the headlines.

**Comment by freckletonj** [comment]
seconded! And thanks, I had no idea why I was getting that weird formatting.

**Comment by integral-dw** [comment]
Hello, I am the current maintainer of this package (more accurately, the version available on MELPA).
I am not sure what exactly you mean, see [here](https://github.com/integral-dw/org-bullets/issues/5).  
Be sure to open a new issue over here [here](https://github.com/integral-dw/org-bullets) for further discussion.

**Comment by freckletonj** [comment]
@integral-dw IIRC there was a conflict with `org-indent`, but... sry, I haven't been able to use `org-bullets` since then.

**Comment by integral-dw** [comment]
I see. If you're interested, try giving `org-superstar` a shot. It's still a rather young package, but it seems to work with `org-indent`.  See the README for more.

**Comment by vcg3rd** [comment]
This seemed like the best place to ask this.  I am having a problem with first level bullets.  Not headers, but actual bullet point. No matter which symbol I use for bullet points (+ or -) the very first bullet shows the character.  The picture explains.

![org-bullets](https://user-images.githubusercontent.com/3229592/84290031-2e6e5d80-ab11-11ea-9f04-b96f18862881.png)

**Comment by integral-dw** [comment]
Hello @vcg3rd, I am the current maintainer of [org-bullets over at MELPA](https://github.com/integral-dw/org-bullets).  Your issue seems very peculiar.  What package are you using, exactly? I am asking because org-bullets should have no capabilities to prettify item bullets (at least not my version of it). If you are using [org-superstar](https://github.com/integral-dw/org-superstar-mode), consider opening a new Issue there.

**Comment by vcg3rd** [comment]
All my Emacs issues are peculiar! :-) I don't program, at all, and just use Emacs for org-mode, so it happens a lot.  Anyway, I realized I was using the code from a[ different org-bullets](https://github.com/sabof/org-bullets), the one your forked, regarding the syntax of the hook, so I switched. Then I thought it might have been a regex snippet[0] about font-lock-add-keywords I picked up on a blog about beautifying Emacs, so I commented it out.

Then I discovered org-superstar and switched to it, and the problem is gone; however, it may well we gone because I commented out the snippet.  At any rate, thanks for your work!  And you can close my issue.

[0] My snippet in case it was the issue and including it helps someone searching. I didn't like it anyway because I didn't understand it.
```
(font-lock-add-keywords 'org-mode
   '(("^ +\\([-*]\\) "
     (0 (prog1 () (compose-region (match-beginning 1) (match-end 1) "•"))))))
```

**Comment by integral-dw** [comment]
Yeah, the snippet is an incomplete implementation of a feature org-superstar supports (that is, prettifying item bullets on top of title ones), so removing that snippet was a good idea.
